src/ExpandObjects/epfilter.f90

```diff
@@ -14391,7 +14393,7 @@ SUBROUTINE CreateNewUnitary
   CALL CreateNewObj('SetpointManager:SingleZone:Cooling')
   CALL AddToObjFld('Name', base + usAirHandlerNameOff,' Economizer Supply Air Temp Manager')
   CALL AddToObjStr('Control Variable','Temperature')
-  CALL AddToObjStr('minimum supply air temperature {C}','13')
+  CALL AddToObjFld('minimum supply air temperature {C}',base + usCoolDesignSupplyTempOff,'')
```

Comment:
So the magic "13" is gone in this line, what about the magic "45" in the following line?